election-api-typescript/src/resultsController.ts

```diff
@@ -0,0 +1,24 @@
+import resultStore from "./resultsService";
+import { Result } from "./types";
+
+const store = resultStore();
+
+const getResult = (id: number): Result | undefined => {
```

Comment:
I know that this is basically a port of the JS version but I'd much prefer to use `null` to indicate missing items than `undefined` if possible.

```suggestion
const getResult = (id: number): Result | null => {
```